C: Basic Output documentation update (#1031)
Codecademy/docs: content/c/concepts/basic-output/terms/putchar/putchar.md

> ## Print Character with `putchar()`

Reviewer: See my comment on the `printf()` term entry.

---
Title: 'putchar()'
Description: 'Print a char in the console.'
Subjects:
  - 'Code Foundations'
  - 'Computer Science'
Tags:
  - 'Char'
  - 'Console'
CatalogContent:
  - 'learn-c'
  - 'paths/computer-science'
---

> The **`putchar()`** function, or "put character" function allow to print a char to `stdout`, normally the console

Reviewer: ```suggestion
The **`putchar()`** function, or "put character" function prints a char to `stdout`, normally the console
```

> The **`putchar()`** function, or "put character" function allow to print a char in the `stdout` file, normally the console

Reviewer: ```suggestion
The **`putchar()`** function, or "put character" function allow to print a char to `stdout`, normally the console
```

## Syntax

```pseudo
putchar(char)
```

> the **`putchar()`** takes a single `char` as an argument and prints it.

Reviewer: ```suggestion
The **`putchar()`** takes a single `char` as an argument and prints it.
```
